lib/cog/adapters/slack/rtm_connector.ex

```diff
@@ -108,7 +108,10 @@ defmodule Cog.Adapters.Slack.RTMConnector do
                               fn() -> handle_command(room, user_id, text, state) end}
                          end
     unless message == nil do
-      Slack.API.send_message(channel, message)
+      {:ok, room} = Slack.API.lookup_room(id: channel)
```

Comment:
I thought about that. But then I thought it would be odd to sometimes have the name and sometimes not have it.